proto/svc-storage-grpc-flight_plan.proto

```diff
@@ -0,0 +1,99 @@
+syntax = "proto3";
+package grpc.flight_plan;
+
+import "google/protobuf/empty.proto";
+import "google/protobuf/field_mask.proto";
+import "svc-storage-grpc.proto";
+
+//FlightPlanRpc service
+service FlightPlanRpc {
+    rpc flight_plans(grpc.SearchFilter) returns (FlightPlans);
+    rpc flight_plan_by_id(grpc.Id) returns (FlightPlan);
+    rpc insert_flight_plan(FlightPlanData) returns (FlightPlan);
+    rpc update_flight_plan(UpdateFlightPlan) returns (FlightPlan);
+    rpc delete_flight_plan(grpc.Id) returns (google.protobuf.Empty);
+}
+
+//Flight Status Enum
+enum FlightStatus {
+    //READY
+    READY = 0;
+    //BOARDING
+    BOARDING = 1;
+    //IN_FLIGHT
+    IN_FLIGHT = 3;
+    //FINISHED
+    FINISHED = 4;
+    //CANCELLED
+    CANCELLED = 5;
+    //DRAFT
+    DRAFT = 6;
+}
+//Flight Priority Enum
+enum FlightPriority {
+    //LOW
+    LOW = 0;
+    //HIGH
+    HIGH = 1;
+    //EMERGENCY
+    EMERGENCY = 2;
+}
+
+//FlightPlan
+message FlightPlan {
+    //id
+    string id = 1;
+    //data
+    FlightPlanData data = 2;
+}
+
+message UpdateFlightPlan {
+    //id
+    string id = 1;
+    FlightPlanData data = 2;
+    google.protobuf.FieldMask mask = 3;
+}
+
+//FlightPlanData
+message FlightPlanData {
+    //pilot_id
+    string pilot_id = 2;
+    //vehicle_id
+    string vehicle_id = 3;
+    //cargo
+    repeated uint32 cargo = 4;
+    /*
+    //weather_conditions
+    string weather_conditions = 5;
+    //departure_vertiport_id
+    string departure_vertiport_id = 6;
+    //pad_id_departure
+    string departure_pad_id = 7;
+    //destination_vertiport_id
+    string destination_vertiport_id = 8;
+    //destination_pad_id
+    string destination_pad_id = 9;
+    //estimated_departure
+    google.protobuf.Timestamp estimated_departure = 10;
```

Comment:
just a heads up, svc-scheduler uses prost_types::Timestamp

They both have a seconds and nanoseconds field, but we probably want to pick one

Reply:
Hmm, good point, I just used what was already there.

Reply:
prost_types is needed in the rust code to match the google.protobuf.Timestamp type.
fixed in https://github.com/Arrow-air/svc-storage/pull/4/commits/e722fb189299be5e2547cac1e5f4310503067828